pyproject.toml

```diff
@@ -23,7 +23,8 @@ ipykernel = "^6.29.5"
 tqdm = "^4.67.0"
 openmm = {extras = ["cuda12"], version = "^8.2.0"}
 openmm-cuda-12 = "^8.2.0"
-atom = { git = "https://github.com/macrocosm-os/atom.git", branch = "main" }
+atom = { git = "https://github.com/macrocosm-os/atom.git", branch = "features/epistula" }
```

Comment:
this can be changed to the main branch and the lock file can be regenerated. I have merged hanging PRs into main on the `atom` side. The major thing that merged was the Pydantic class checking for the epistula request, so make sure to retest this